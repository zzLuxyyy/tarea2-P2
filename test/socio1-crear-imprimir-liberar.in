crearSocio 12345678 Pedro Alfonso 4 6 2000
imprimirCISocio
imprimirNombreYApellidoSocio
imprimirSocio
imprimirFechaAltaSocio
liberarSocio

crearSocio 23429221 Luis Rodriguez 9 2 1998
imprimirCISocio
imprimirNombreYApellidoSocio
imprimirSocio
imprimirFechaAltaSocio
liberarSocio

crearSocio 98766543 Mateo Perez 29 9 1977
imprimirCISocio
imprimirNombreYApellidoSocio
imprimirSocio
imprimirFechaAltaSocio
liberarSocio

crearSocio 23429221 Jose Riquelme 6 11 1986
imprimirCISocio
imprimirNombreYApellidoSocio
imprimirSocio
imprimirFechaAltaSocio
liberarSocio

crearSocio 55549321 Leopoldo Suarez 23 1 2002
imprimirCISocio
imprimirNombreYApellidoSocio
imprimirSocio
imprimirFechaAltaSocio
liberarSocio

crearSocio 44199264 Mirian Lopez 14 7 1980
liberarSocio
Fin
