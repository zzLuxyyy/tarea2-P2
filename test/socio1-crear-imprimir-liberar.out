1>Socio de ci 12345678 creado exitosamente.
2>La CI del socio es: 12345678
3>El nombre del socio es: Pedro
El apellido del socio es: Alfonso
4>Socio 12345678: Pedro Alfonso
Fecha de alta: 4/6/2000
Géneros favoritos: 
5>La fecha de alta del socio es: 4/6/2000
6>Se ejecutó 'liberarTSocio' exitosamente.
7>Socio de ci 23429221 creado exitosamente.
8>La CI del socio es: 23429221
9>El nombre del socio es: Luis
El apellido del socio es: Rodriguez
10>Socio 23429221: Luis Rodriguez
Fecha de alta: 9/2/1998
Géneros favoritos: 
11>La fecha de alta del socio es: 9/2/1998
12>Se ejecutó 'liberarTSocio' exitosamente.
13>Socio de ci 98766543 creado exitosamente.
14>La CI del socio es: 98766543
15>El nombre del socio es: Mateo
El apellido del socio es: Perez
16>Socio 98766543: Mateo Perez
Fecha de alta: 29/9/1977
Géneros favoritos: 
17>La fecha de alta del socio es: 29/9/1977
18>Se ejecutó 'liberarTSocio' exitosamente.
19>Socio de ci 23429221 creado exitosamente.
20>La CI del socio es: 23429221
21>El nombre del socio es: Jose
El apellido del socio es: Riquelme
22>Socio 23429221: Jose Riquelme
Fecha de alta: 6/11/1986
Géneros favoritos: 
23>La fecha de alta del socio es: 6/11/1986
24>Se ejecutó 'liberarTSocio' exitosamente.
25>Socio de ci 55549321 creado exitosamente.
26>La CI del socio es: 55549321
27>El nombre del socio es: Leopoldo
El apellido del socio es: Suarez
28>Socio 55549321: Leopoldo Suarez
Fecha de alta: 23/1/2002
Géneros favoritos: 
29>La fecha de alta del socio es: 23/1/2002
30>Se ejecutó 'liberarTSocio' exitosamente.
31>Socio de ci 44199264 creado exitosamente.
32>Se ejecutó 'liberarTSocio' exitosamente.
33>Fin.
